libs/service-portal/health/src/components/ExtraDoctors/index.tsx

```
@@ -0,0 +1,26 @@
+
+export const ExtraDoctors = ({ doctors }: Props) => {
+  const docs = doctors?.map((doctor) => doctor.name)
+
+  if (!docs || docs.length === 0) {
+    return null
+  }
+
+  return (
+    <Box>
+      {docs.map((doc) => (
+        <Box>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)